kafka_consumer_freshness_tracker/src/main/java/com/tesla/data/consumer/freshness/ConsumerFreshness.java

```diff
@@ -281,7 +283,37 @@ private ListenableFuture<String> measureCluster(Burrow.ClusterClient client) {
     // if all the consumer measurements succeed, then we return the cluster name
```

Comment:
`if all the consumer measurements succeed ` -> `if at least one consumer measurement succeeds`?